worlds/animal_well/bean_patcher.py

```diff
@@ -303,6 +314,9 @@ def __init__(self, process=None, logger=None):
         self.tracker_goal: str = ""
         self.tracker_initialized = False
 
+        self.save_file: str = None
+        self.save_seed: str = None
```

Comment:
```suggestion
        self.save_file: Optional[str] = None
        self.save_seed: Optional[str] = None
```
Doesn't fix the issue, but might as well do this now instead of a separate commit later